lib/dialog/style.scss

```diff
@@ -39,6 +47,12 @@
   display: flex;
   flex-direction: column;
   flex: 1 0 auto;
+  max-height: $dialog-max-height - $dialog-title-height;
+  overflow: auto;
+
+  @media only screen and (max-height: $dialog-max-height) {
+    max-height: calc(100vh - 2rem - #{$dialog-title-height});
```

Comment:
I'm not yet had to use Interpolation in Sass, is it needed here? I believe as this is a variable it should work without just as `max-height: calc(100vh - 2rem - $dialog-title-height);`

Reply:
I checked the [documentation for this function](https://sass-lang.com/documentation/syntax/special-functions#calc-clamp-element-progid-and-expression) and looks like that interpolation is required here for using the `calc` function from CSS:
>The calc(), clamp() and element() functions are defined in the CSS spec. Because calc()’s mathematical expressions conflict with Sass’s arithmetic, and element()’s IDs could be parsed as colors, they need special parsing.

In fact if I remove the interpolation, it produces an invalid CSS property (see attached screenshot).

<img width="306" alt="Screenshot 2021-03-31 at 18 15 38" src="https://user-images.githubusercontent.com/14905380/113176855-37215300-924d-11eb-8100-5a86cf4d6d4c.png">

Reply:
Ah thank you! I had tested that and everything looked good in the devtools to me, but I must have been looking at the second rule there and missed the first that has the error. 
All looks good to me.